Sources/SwiftFrameCore/Extensions/Dictionary+Extensions.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+public extension Dictionary {
+
+    subscript(safe key: Key?) -> Value? {
```

Comment:
Since this is only used in one place, you could maybe replace it with the same `flatMap` trick mentioned earlier